core-bundle/src/Controller/AbstractFragmentController.php

```diff
@@ -56,21 +55,6 @@ public static function getSubscribedServices(): array
         return $services;
     }
 
-    protected function getPageModel(): PageModel|null
```

Comment:
The changes in this file I would extract into its own PR because this cleanup always makes sense, no matter this PR :)